src/utilities/util_load_json.c

```diff
@@ -559,24 +559,3 @@ json_skip(WT_SESSION *session, JSON_INPUT_STATE *ins, const char **matches)
  * util_load_json --
  *     Load from the JSON format produced by 'wt dump -j'.
  */
-/*
```

Comment:
Was the intended to be removed? If yes should we remove the header of the function as well?